### DISCUSSION TITLE
Cannot Show non-Western Characters in FreePlane tab

### DISCUSSION THREAD

**Discussion by yasenstar** [freeplane/freeplane#2126] [Help]
Hello,

I have created mindmap in Chinese character in the root node, the display in the tab is not recognizable.

![freeplane-tab-non-western-character](https://github.com/user-attachments/assets/71f1aeef-8237-4245-bdf9-e49633d5f3a2)

I think this is the common issue for non-Western characters, is there any setting that I can change to cope with such kind of charset, e.g. UTF-8?

Regards, Xiaoqi

**Comment by fwfsf** [comment]
Try setting the appearance as default, some other appearance settings may lose CJK fonts.

![fsweidsewsdj](https://github.com/user-attachments/assets/18bebaaf-4121-4dfe-879c-8e8a39fcf23f)
